### PR TITLE
Added utf-8 support for ftp storage

### PR DESCRIPTION
**Pull request by AlexElizard**
Please add utf-8 support for ftp storage .

### CODE REVIEW

**Comment by jschneier** [comment]
I see, the default is `latin-1`. Can we add this as a setting?

**Comment by AlexElizard** [comment]
Yes. Only in the form of setting in the settings.py file, please. Thanks

**Comment by jschneier** [comment]
Please update the PR with the setting.

File: storages/backends/ftp.py [hunk 1]
```diff
@@ -41,13 +41,14 @@ class FTPStorageException(Exception):
...
 
-    def __init__(self, location=None, base_url=None):
+    def __init__(self, location=None, base_url=None, encoding=None):
         location = location or setting('FTP_STORAGE_LOCATION')
         if location is None:
             raise ImproperlyConfigured("You must set a location at "
                                        "instanciation or at "
                                        " settings.FTP_STORAGE_LOCATION'.")
         self.location = location
+        self.encoding = encoding or settings('FTP_STORAGE_ENCODING') or 'latin-1'
```
Comment (jschneier) [inline storages/backends/ftp.py]: This should be `setting(` instead of `settings(`.

Comment (AlexElizard) [reply storages/backends/ftp.py]: Thanks. Fixed

**Pull request merged**